src/tools/rbd/action/Export.cc

```diff
@@ -580,7 +598,8 @@ static int do_export(librbd::Image& image, const char *path, bool no_progress,
   uint64_t period = image.get_stripe_count() * (1ull << info.order);
 
   if (export_format == 1)
-    r = do_export_v1(image, info, fd, period, max_concurrent_ops, pc);
+    r = do_export_v1(image, info, fd, period, max_concurrent_ops, pc,
+                     win32_phys_disk);
   else
     r = do_export_v2(image, info, fd, period, max_concurrent_ops, pc);
```

Comment:
Indeed, I was thinking about only covering ``export_format == 1`` for the time being. We might need some padding for ``export_format == 2`` around the encoded image metadata because of the sector size constraints.

Reply:
Only covering v1 for the time being makes sense.  We need a error out early and with a clear error message in the v2 case is all I'm saying.